changelog/2.076.0_pre.dd

```diff
@@ -0,0 +1,374 @@
+$(LI $(BUGZILLA 12866): Append to std.container.Array of fixed size arrays)
+$(LI $(BUGZILLA 16079): memoize should cache objects too)
+$(LI $(BUGZILLA 16191): std/digest/digest.d should be renamed to package.d)
+$(LI $(BUGZILLA 17389): std.traits.hasNested fails for class with member of same type)
+$(LI $(BUGZILLA 17458): [std.regex] Assertion failure in package.d)
+$(LI $(BUGZILLA 17539): std.parallellism.parallel triggers 'statement not reachable' warnings in the rdmd_test suite)
+$(LI $(BUGZILLA 17562): Tangent function returns NaN for abs$(LPAREN)input$(RPAREN) >= 2^63)
+$(LI $(BUGZILLA 17574): Range violation in std.getopt:getopt AA parsing)
+$(LI $(BUGZILLA 17616): makeIndex cannot fully use range of index type)
+$(LI $(BUGZILLA 17640): std.concurrenct writeln conflicts with std.stdio writeln in unittests)
+$(LI $(BUGZILLA 17711): std.array.byPair should be usable with const AA)
+)
+$(BUGSTITLE Phobos enhancements,
+
+$(LI $(BUGZILLA 2137): Data not compressed on fly when adding to zip archive)
+$(LI $(BUGZILLA 6384): std.traits.isComparable)
+$(LI $(BUGZILLA 15759): chunks should work with only an input range)
+$(LI $(BUGZILLA 15771): FileLogger should create the output directory if it does not exist)
+$(LI $(BUGZILLA 16062): Add 'clear' method to OutBuffer $(LPAREN)std.outbuffer$(RPAREN))
+$(LI $(BUGZILLA 16744): We should have a TypeOf template so that typeof can be used with templates like staticMap)
+$(LI $(BUGZILLA 16993): Documentation for toSimpleString and toString does not explain how they differ)
+$(LI $(BUGZILLA 17250): ProcessPipes $(LPAREN)std.process$(RPAREN) should provide a test for a null pid)
+$(LI $(BUGZILLA 17365): https://dlang.org/phobos/std_experimental_checkedint.html is missing the Throw hook description)
+$(LI $(BUGZILLA 17369): [Module std.traits] Documentation lists ditto in table)
+$(LI $(BUGZILLA 17525): std.algorithm.searching.skipOver should have a single argument with pred version)
+$(LI $(BUGZILLA 17540): std.net.curl: HTTP no possibillity to set CURLOPT_NOPROXY)
+$(LI $(BUGZILLA 17587): JSONOptions: No forward slash encoding)
+)
+$(BUGSTITLE Druntime regressions,
+
+$(LI $(BUGZILLA 15619): [REG 2.066] Floating-point x86_64 codegen regression, when involving array ops)
+)
+$(BUGSTITLE Druntime bugs,
+
+$(LI $(BUGZILLA 11594): synchronized causing segfault instead of Error.)
+$(LI $(BUGZILLA 14319): core.demangle does not support member function attributes)
+$(LI $(BUGZILLA 14563): core.demangle: Does not demangle type modifers)
+$(LI $(BUGZILLA 14576): [ddemangle] core.demangle unable to handle ambiguity in symbols)
+$(LI $(BUGZILLA 16856): D does not work on FreeBSD current $(LPAREN)what will eventually be 12$(RPAREN) due to libunwind)
+$(LI $(BUGZILLA 17609): core.demangle demangles delegate variables as functions)
+$(LI $(BUGZILLA 17610): core.demangle shows return type of template alias parameter)
+$(LI $(BUGZILLA 17611): core.demangle cannot demangle delegates with function attributes)
```

Comment:
Do these mangling related bug fixes depend on Rainer's unmerged dmd PR, or the merged druntime PR is sufficient?